README.md

```diff
@@ -35,6 +35,13 @@ Each modification requires a new build, here is how to run a build:
 
     cake build
 
+### To hack cozy-proxy using the cozy vagrant
+
+- Forward a new port from your vagrant (for example : `config.vm.network :forwarded_port, guest: 9555, host: 9555`)
```

Comment:
* typo: "for example: " (no space before ":" in english)
* I suggest: Forward a new port from the virtual machine (for example: `config.vm.network :forwarded_port, guest: 9555, host: 9555` in file Vagrantfile)